lua/cmp/view/ghost_text_view.lua

```diff
@@ -51,6 +47,20 @@ ghost_text_view.new = function()
   return self
 end
 
+---Generate the ghost text (characters not bytes)
+ghost_text_view.text_gen = function(self, line, cursor_col)
+  local text
+  local cword = string.sub(line, self.entry:get_offset(), cursor_col)
+  local cword_char_len = vim.fn.strchars(cword)
+  local entry_text = self.entry:get_insert_text()
+  if self.entry.completion_item.insertTextFormat == types.lsp.InsertTextFormat.Snippet then
+    entry_text = vim.lsp.util.parse_snippet(entry_text)
+  end
+  -- Missing characters to complete the entry text
+  text = vim.fn.strcharpart(str.oneline(entry_text), cword_char_len)
```

Comment:
Could you use `vim.str_byteidx` instead of `vim.fn.strcharpart`?

Reply:
Done